content/guides/antora.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Antora
+description: Build your website with the Antora Static Site Generator (SSG) and host it on Clever Cloud. No dedicated runner needed.
+tags:
+- guides
+keywords:
+- antora
+- asciidoc
+- nodejs
+- static
+- ssg
+- html
+- js
+- css
+- website
+type: "docs"
+comments: false
+draft: false
+---
+[Antora](https://antora.org/) is an [AsciiDoc](https://asciidoc.org/) based Static Site Generator focused on technical documentation writing. It promotes *docs as code*, using a YAML playbook and git repositories to organize and host content. If you need an example source code, use the [demo playbook](https://docs.antora.org/antora/latest/install-and-run-quickstart/#create-a-playbook) (you'll need [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Node.js](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs)):
+```bash
+git init
```

Comment:
Why do we init a `.git` repository outside the app?

Reply:
Why outside the app ? It's the initial `git init` we'll need later to commit/deploy. 

As mentioned here, this guide/example is based on the default playbook provided in the Antora documentation as an example.